File: loop5.cpp
#include <iostream>
using namespace std;

int main()
{
    int n, rev = 0;

    cout << "Enter a number: ";
    cin >> n;

    for(; n > 0; n = n / 10)
    {
        int digit = n % 10;

        rev = rev * 10 + digit;
    }

    cout << "Reverse = " << rev << endl;

    return 0;
}

File: loop5.java
public class loop5 {
    public static void main(String[] args) {
        


        int i  = 3749;
        int rev = 0;
        for(;i>0; i =i/10) {
   int num = i %10;
  rev = rev* 10 + num;
        }
    
     
        System.out.println("Reverse = " + rev);
    }
}
